src/views/Farm/components/HarvestLPsNoticeYAMYUSD.tsx

```diff
@@ -36,7 +36,7 @@ const HarvestLPsNoticeYAMYUSD: React.FC = () => {
       return (
         <>
           <Notice>
-            <NoticeIcon>❗</NoticeIcon>
+            <StyledNoticeIcon role='img'>❗</StyledNoticeIcon>
```

Comment:
any reason for the redundency? this can be one component `StyledNoticeIcon` to be used everywhere